### PR TITLE
chore: call breaker_cb_ on shutdown

### PR DESCRIPTION
**Pull request by romange**

### CODE REVIEW

**Review by dranikpg — DISMISSED** [review]
🤷🏻‍♂️

File: src/facade/dragonfly_connection.cc [hunk 1]
```diff
@@ -1718,6 +1698,15 @@ void Connection::DecreaseStatsOnClose() {
   --stats_->num_conns;
 }
 
+void Connection::BreakOnce(uint32_t ev_mask) {
+  if (breaker_cb_) {
+    DVLOG(1) << "[" << id_ << "] Connection::breaker_cb_ " << ev_mask;
+    auto fun = std::move(breaker_cb_);
+    DCHECK(!breaker_cb_);
+    fun(ev_mask);
```
Comment (dranikpg) [inline src/facade/dragonfly_connection.cc:1706]: `std::exchange(breaker_cb_, nullptr)()` 😎

**Review by dranikpg — APPROVED** [review]
Ok, that's at least safer 👍🏻

File: src/facade/dragonfly_connection.cc (hunk 1 above)
Comment (romange) [reply src/facade/dragonfly_connection.cc:1706]: :)

**Pull request merged**